osfoffline/utils/__init__.py

```diff
@@ -1,6 +1,9 @@
 import hashlib
 import os
 
+from PyQt5 import QtCore
+from PyQt5.QtCore import QCoreApplication
+from PyQt5.QtWidgets import QApplication
```

Comment:
By more UI-centric do you mean inside the `preferences.py` or in something like a `osfoffline/utils.py`?